src/components/index.d.ts

```diff
@@ -542,47 +542,47 @@ export declare const ButtonSelectGroup: {
     ...rest
   }: {
     [x: string]: any
-    labelCopy: any
+    labelCopy: string
     children: any
-    initialValue?: any
-    currentValue: any
-    currentError: any
-    formTouched: any
-    onSelect: any
-    column: any
-    formChangeHandler: any
+    initialValue?: string | boolean
+    currentValue?: string | boolean
+    currentError?: string
+    formTouched?: boolean
+    onSelect?: any
```

Comment:
On later iterations I'll deal with typing functions as I'll probably want to create interfaces and what not.